packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -77,6 +77,7 @@ export default function Preview( { idBase, instance, isVisible } ) {
 				</Placeholder>
 			) }
 			<div
+				data-widget-preview-id={ idBase }
```

Comment:
I think `idBase` is just something like `block` and `search`, maybe we should use `id` from the parent component?

Reply:
It depends on what we're after. leaving it like this should make stylinh easier:

<img width="357" alt="Screen Shot 2021-07-13 at 19 15 34" src="https://user-images.githubusercontent.com/107534/125488295-287f772a-7ab2-4a52-9ef1-e7685dbed892.png">

Two categories widgets and same `data-widget-preview-id` so one CSS rule. I think?

Reply:
They can always use `[data-widget-preview-id^="categories"]` to target all the categories previews though.

Reply:
Why not just something like `data-widget-preview-type`? I agree calling it ID is strange. ID indicates a unique identifier, and this is not unique.

Reply:
Also, would it make sense to include the type higher up on the `.wp-block` element instead?

Reply:
`data-widget-id-base` would be fine to me. We call this `id_base` in a bunch of places including the widgets REST API.

Reply:
This change doesn't fix https://github.com/WordPress/gutenberg/issues/32805. See https://github.com/WordPress/gutenberg/issues/32805#issuecomment-880376265. I think that we should not include it while we think of an actual solution.